ziti/cmd/ziti/constants/constants.go

```diff
@@ -36,8 +36,8 @@ const (
 
 // Config Template Constants
 const (
-	DefaultListenerBindPort  = 10080
-	DefaultGetSessionTimeout = 60 * time.Second
+	DefaultZitiEdgeRouterListenerBindPort = "10080"
```

Comment:
It was mainly for future-proofing and to be more in line with the other environment variables, in case there was ever more than one type of router listener port. The environment variable that represents this value (when the default isn't used) is `ZITI_EDGE_ROUTER_LISTENER_BIND_PORT` and most other values in the portion of the config this is used start with `ZITI_EDGE_ROUTER_LISTENER...`

Reply:
Would you prefer changing the Go variable, both, or something else.

Again, I changed it only to resemble the env var that it represents, previously there was not env var which is why it was much more succinct.